core/src/main/resources/reference.conf

```diff
@@ -434,6 +434,9 @@ database {
   #  connectionTimeout = 5000
   #}
 
+  # For batch inserts the number of inserts to send to the DB at a time
+  # insert-batch-size = 2000
```

Comment:
ToL: This is actually a bit of a misnomer since it only applies to metadata inserts